### PR TITLE
Setup GitHub pages

### PR DESCRIPTION
**Pull request by Uddekudde**
### Why

- We need to host the project on github pages.

### What

- Added homepage, scripts and gh-pages dependency.

### How 

- See that the deploy command deploys the project to github pages correctly

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +13,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
+	"predeploy": "npm run build",
```
Comment (phek) [inline package.json]: What is the `predeploy` script used for? (:

If this script is required we should use `predeploy: "yarn build"` instead now when we're using yarn

Comment (Uddekudde) [reply package.json]: The predeploy runs first when deploy is run. 
Right! I missed that.

**Comment by Uddekudde** [comment]
> When I go to https://uddekudde.github.io/tudd-pekblom/ I only see the README. Do we need to do something to see the app?

This was due to the master branch being selected for github pages in the project settings, its's fixed now.

**Comment by Uddekudde** [comment]
Fixed predeploy script in: 9214f4bbfc56eb9163f24e663f57f88d21b371c4

**Pull request merged**